packages/fcl-react-native/src/utils/react-native/render-deeplink.js

```diff
@@ -1,6 +1,6 @@
 import * as Linking from "expo-linking"
 import {AppState} from "react-native"
-import {URL} from "../url"
+import {URL} from "@onflow/fcl-core"
```

Comment:
I think our URL patch might be able to be replaced by adding `react-native-url-polyfill` as a dependency in fcl-react-native now that we made the split actually.  But not a big deal, we can follow this stuff up with another PR with the rest of the react native changes imo